src/App/Documentation/components/Toast/index.js

```diff
@@ -16,14 +15,15 @@ const Overview = () => (
             {"<button onclick=\"px.toast({ html: 'I am a toast!' })\" class=\"btn btn-primary\">Toast!</button>"}
         </ComponentPreview>
         <ComponentPreview language="html" showCasePanel>
-            <button className="btn btn-primary" type="button" onClick={() => toast({ html: "I am a toast!" })}>Click for toast!</button>
+            <button className="btn btn-primary" type="button" onClick={() => px.toast({ html: "I am a toast!" })}>Click for toast!</button>
```

Comment:
Why do we use inline JS in our examples?

Reply:
I think it is just to show that we use our script to bind a click event to the button. @erikhallaraaker ?